charts/spire/charts/spire-server/templates/configmap.yaml

```
@@ -1,4 +1,4 @@
-{{ $namespace := .Release.Namespace }}
+{{ $server := . }}
```

```suggestion
{{ $root := . }}
```

Is a common convention, as you refer to the root of the values. $server is misleading, as `.` refers to root of values.